lang/clojure/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 
 name                clojure
 epoch               20110929
-version             1.10.1.754
+version             1.10.1.763
+revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
I pushed update. I didn't know i should reset the `revision` when increasing `version` field. Sorry about that.